DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -44,6 +46,12 @@ FROM Tutorials
                          INNER JOIN CustomisationTutorials
                          ON CustomisationTutorials.TutorialID = Tutorials.TutorialID
 
+                         INNER JOIN Customisations
+                         ON CustomisationTutorials.CustomisationID = Customisations.CustomisationID
+
+                         INNER JOIN Applications
+                         ON Customisations.ApplicationId = Applications.ApplicationId
+
                          INNER JOIN Progress
                          ON aspProgress.ProgressID = Progress.ProgressID
                             AND CustomisationTutorials.CustomisationID = Progress.CustomisationID

```

Comment:
One thing I just noticed, our WHERE statement is a bit different to the one in uspReturnProgressDetail_V3. In particular we're not checking the CustomisationTutorials.Status and not checking Customisation.Active. We should update that to match the stored procedure, as we don't want to return different results than the stored procedure 🙂

Reply:
Thank you for catching that! Added in 86f7166 🙂